charts/aws-vpc-cni/templates/daemonset.yaml

```diff
@@ -128,6 +128,8 @@ spec:
             - --enable-ipv6={{ .Values.nodeAgent.enableIpv6 }}
             - --enable-network-policy={{ .Values.enableNetworkPolicy }}
             - --enable-cloudwatch-logs={{ .Values.nodeAgent.enableCloudWatchLogs }}
+            - --metrics-bind-addr={{ .Values.nodeAgent.metricsAddr }}
```

Comment:
Minor nit: maybe `metricsAddr` -> `metricsBindAddr` since it will be consistent with other config naming

Reply:
updated